nautobot/cloud/templates/cloud/cloudnetwork_retrieve.html

```diff
@@ -85,22 +120,9 @@
     <div class="panel panel-default">
         <div class="panel-heading">
             <strong>Extra Config</strong>
-            {% include 'extras/inc/json_format.html' %}
         </div>
         <div class="panel-body">
-            {% include 'extras/inc/json_data.html' with data=object.extra_config format=format %}
+            <pre>{{ object.extra_config|render_json }}</pre>
```

Comment:
Why are we making this change?

Reply:
Consistency with other templates perhaps? I'm not familiar with what `json_data.html` provides us, TBH.

Reply:
- The json/yaml buttons didn't work when I tested it
- Large data structures were being rendered as a single long line instead of formatted with line breaks and indents

Reply:
From next.demo:

![image](https://github.com/user-attachments/assets/480d11f6-72c5-4e97-92bb-10ad8c09f112)

When you click on the JSON or YAML buttons:

![image](https://github.com/user-attachments/assets/a81e90f5-aeea-4b15-9aaa-30d0bef5adee)

When you click

Reply:
Interesting... We use the pattern of json/yaml in other places that have the same model field. I wonder why this one sometimes fails. I personally use the yaml side more often, and I default my views to show the yaml.

Reply:
I think we were missing the handling of the `format` query parameter in the view to make this work. See the implementation for ConfigContext [here](https://github.com/nautobot/nautobot/blob/0f0b2b166629d25554f04640a71216d30d9055bb/nautobot/extras/views.py#L192-L211)

Reply:
I will create a followup ticket to add this back in with the `format` query parameter.